DAMPER_STEEPNESS: 5
MANIPULABILITY_THRESHOLD: 0.07
CONTROL_RATE: 100
TWIST_GAIN: 
  - 0.1
  - 0.1
DRIFT_GAIN: 
  p: 
    linear: 2
    angular: 8
  d:
    linear: 1
    angular: 2

SAMPLE_STATES:
  r: [-0.6981317007977318, 0.5235987755982988, -1.5707963267948966, -1.5882496193148399, -0.5410520681182421, -0.6806784082777885, 1.5882496193148399]
  l: [0.6806784082777885, 0.47123889803846897, 1.5882496193148399, -1.6057029105859523, 0.5410520681182421, -0.6632251157578453, 1.4835298641951802]

JOINT_NAMES :
  left: 
    - l_shoulder_pan_joint
    - l_shoulder_lift_joint
    - l_upper_arm_roll_joint
    - l_elbow_flex_joint
    - l_forearm_roll_joint
    - l_wrist_flex_joint
    - l_wrist_roll_joint
  right:
    - r_shoulder_pan_joint
    - r_shoulder_lift_joint
    - r_upper_arm_roll_joint
    - r_elbow_flex_joint
    - r_forearm_roll_joint
    - r_wrist_flex_joint
    - r_wrist_roll_joint

